doc/specifications/rest/REST-API-V2.rst

```diff
@@ -1454,6 +1455,24 @@ Delete Content Version
     :401: If the user is not authorized to delete this version
     :403: If the version is in state published
 
+Delete Content Version Translation
+``````````````````````
+:Resource: /content/objects/<ID>/versions/<versionNo>/translations/<language_code>
+:Method: DELETE
+:Description: The translation is removed from a draft
+:Headers:
+        :X-Set-Main-Translation: [language_code] to be set as main translation (useful if deleting main translation)
```

Comment:
and if I'm not deleting the main translation, does this still set the main translation ?
also, if that's optional, that means you need to implement a "main translation fallback" mechanism anyway.
In addition, from a PlatformUI perspective, this header will never be set unless we ask the user which translation is the main one but that's not how the feature is specified.

So I'd rather keep things "simple" by not having such header (at least in a first version).

Reply:
Ok, so the only possibility here is for REST endpoint to decide on new `mainLanguageCode` if the removed language is the main one, right?

My idea here was to allow editor to make this choice if needed, but you're right - this is not within the scope of this feature.